src/github.js

```diff
@@ -112,6 +112,18 @@ async function assign_reviewers(reviewers) {
   });
 }
 
+// https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members
+async function get_team_members(team) {
+  const context = get_context();
+  const octokit = get_octokit();
+
+  console.log(context);
+  return octokit.teams.listMembersInOrg({
+    org: context.repo.org,
+    team_slug: team,
+  });
+}
```

Comment:
[Note] Example response

```
[
  {
    "login": "octocat",
    "id": 1,
    "node_id": "MDQ6VXNlcjE=",
    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
    "gravatar_id": "",
    "url": "https://api.github.com/users/octocat",
    "html_url": "https://github.com/octocat",
    "followers_url": "https://api.github.com/users/octocat/followers",
    "following_url": "https://api.github.com/users/octocat/following{/other_user}",
    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
    "organizations_url": "https://api.github.com/users/octocat/orgs",
    "repos_url": "https://api.github.com/users/octocat/repos",
    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
    "received_events_url": "https://api.github.com/users/octocat/received_events",
    "type": "User",
    "site_admin": false
  }
]
```

Reply:
Since `src/github.js'` is a wrapper, it would make more sense to return a simple array like `[user1, user,2, etc.]`. We can add something like `map((member) => member.login)` here.